vision_agent/tools/tools.py

```diff
@@ -2162,6 +2194,7 @@ def _plot_counting(
     generate_pose_image,
     closest_mask_distance,
     closest_box_distance,
+    docqa_image,
```

Comment:
remove from FUNCTION_TOOLS for now

Reply:
Done, sorry, I left it there while testing.